lib/msf/core/post/windows/packrat.rb

```diff
@@ -146,7 +145,7 @@ def extract_json(saving_path, artifact_child, artifact, local_loc)
 
           child_json_query.each do |split|
             children = eval("json_parse#{parent_json_query}")
-            children.each do |child_node|
+            children.each do |_child_node|
               child = eval("child_node#{split}").to_s
               json_credential_save << "#{split}:  #{child}"
             end
```

Comment:
cc @bwatters-r7 - I don't suppose you have any context on whether `child` meant to be `child_node` here? 👀 
Not sure if this was a codepath which was tested as part of the larger packrat effort, happy to change it over though

Reply:
I think its using `eval` to split the contents of `child_node` and then save it into the `child` variable? Weird naming of variables but I think the logic is right here unless I'm missing something? This might also explain why renaming this `_child_node` might not work?

Just a passing thought/remark though, its entirely possible I misread this.

Reply:
Confirmed this was an issue, and is indeed relying on this to evaluate a raw ruby string which relies on `child_node` being present

```
[*] undefined local variable or method `child_node' for #<Module:post/windows/gather/credentials/seamonkey datastore=[{"WORKSPACE"=>nil, "VERBOSE"=>false, "SESSION"=>nil, "REGEX"=>"^password", "STORE_LOOT"=>true, "EXTRACT_DATA"=>true, "ARTIFACTS"=>"All"}]>
```

Will fix.